### ISSUE TITLE
Direct cluster apply

### ISSUE THREAD

**Issue by RealityAnomaly** [RealityAnomaly/architect#10]
Provide a mechanism for applying resources directly to the cluster via the Kubernetes API. We need to have a mechanism for waiting for resources to become ready, like FluxCD does, to avoid problems, and proper dependency management.

**Comment by RealityAnomaly** [comment]
direct apply is now supported, waiting on ready will be a separate issue
